src/Widget/WidgetPageInterface.php

```diff
@@ -170,6 +170,20 @@ public function setCss($css);
      */
     public function getCss();
 
+    /**
+     * Set the selected theme
+     *
+     * @param string $selectedTheme
+     */
+    public function setSelectedTheme($selectedTheme);
+
+    /**
+     * Get the selected theme.
+     *
+     * @return string
+     */
+    public function getSelectedTheme();
```

Comment:
I would suggest removing the comment block and declaring the return type of the function. 

This would give the same amount of information (and would also make sure the return type is `string`) but instead of comments, you use language constructs.